CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# code owners are automatically assigned to review PRs
+
+# The goal is to not have more than 2 people responsible to avoid having too
+# many reviewers which often leads to nobody feeling responsible to actually
+# perform the review.
+
+dependencies/non-integer/ @mgudemann
+docs/delegation_design_spec/ @dnadales @kantp
```

Comment:
I guess @JaredCorduan has a better understanding of the delegation design spec, since he implemented operational rules for it :) Does it make sense to have him as owner?

Reply:
sure why not, do you agree @JaredCorduan  ?

Reply:
I'm happy to be a code owner of this document!